tests/unit_test/tool/job/config/configer_test.py

```diff
@@ -34,23 +40,20 @@
     (
         "launch_once",
         [
-            ["app/config_fed_client.conf", "app_script=cifar10_fl.py", "launch_once=false"],
+            ["config_fed_client.conf", "app_script=cifar10_fl.py", "launch_once=False"],
```

Comment:
good we have a new case without "app",
can we keep the old case that tested both Upper case and Lower case for "false"?